.github/actions/setup-postgres-wallet-plugin/action.yml

```diff
@@ -10,6 +10,7 @@ runs:
         sudo apt-get install -y libzmq3-dev libsodium-dev pkg-config libssl-dev
         curl https://sh.rustup.rs -sSf | bash -s -- -y
         export PATH="/root/.cargo/bin:${PATH}"
+        rustup default 1.63.0
```

Comment:
Can you add a comment to both files with the reason of pinning to this rust version with a link to the issue?